backend/main.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/Azure/ARO-HCP/internal/database"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
```

Comment:
Here's the source of the lint error.  These need to be moved to the preceding group of imports, alongside `ocmsdk`.

Reply:
thanks! I was too lazy to run golangci-lint locally :stuck_out_tongue: